rotkehlchen/api/v1/fields.py

```diff
@@ -66,6 +67,56 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
+class IncludeExcludeListField(fields.Field):
+    """ A field that accepts an object like the following and deserializes it to the proper types.
+    {
+        "values": [val1, val2, val3],
+        "behaviour": "exclude"
+    }
+    Eventually, this object will be used to create a filter query to the db, to return all the db
+    table entries where this field's value is not in [val1, val2, val3]. The behaviour key
+    can be either "include" or "exclude" and it defaults to "include" if not present.
+    """
+    def __init__(
+            self,
+            values_field: 'SerializableEnumField',
+            *args: Any,
+            **kwargs: Any,
+    ) -> None:
+        super().__init__(*args, **kwargs)
+        self.values_field = values_field
+
+    def _deserialize(
+            self,
+            value: dict[str, Union[list, str]],
+            attr: Optional[str],
+            data: Any,
+            **kwargs: Any,
+    ) -> IncludeExcludeFilterData:
+
+        if 'behaviour' not in value:
+            # If for any reason the behaviour key is not given, we use the default behaviour.
+            value['behaviour'] = 'include'
+
+        values = value['values']
+        behaviour = value['behaviour']
```

Comment:
shouldn't you check that 'behaviour' is given as a key? Either with an `in` or a try/except KeyError?

Reply:
I check if it is not given, and add the default value if not given:
```
        if 'behaviour' not in value:
            # If for any reason the behaviour key is not given, we use the default behaviour.
            value['behaviour'] = 'include'
```

Reply:
Ah sorry Dimitris. I meant the other one. `value["values"]`